tests/conftest.py

```diff
@@ -197,9 +199,8 @@ def basic_image_file(tmpdir, basic_image):
         "height": image.shape[0],
         "nodata": None
     }
-    with rasterio.drivers():
-        with rasterio.open(outfilename, 'w', **kwargs) as out:
-            out.write(image, indexes=1)
+    with rasterio.open(outfilename, 'w', **kwargs) as out:
+        out.write(image, indexes=1)
```

Comment:
This change is something I've done throughout the tests: removing outer environments where we don't need them.